apps/verification/src/verification.service.ts

```diff
@@ -726,109 +732,144 @@ export class VerificationService {
         });
       }
 
-      const requestedAttributes = getProofPresentationById?.response?.request?.indy?.requested_attributes;
-      const requestedPredicates = getProofPresentationById?.response?.request?.indy?.requested_predicates;
-      const revealedAttrs = getProofPresentationById?.response?.presentation?.indy?.requested_proof?.revealed_attrs;
+      const extractedDataArray: IProofPresentationDetails[] = [];
 
+      if (getProofPresentationById?.response?.request?.presentationExchange) {
```

Comment:
Add comment for for which type fetching details

Reply:
Resolved